CHANGELOG.md

```diff
@@ -1,3 +1,80 @@
+3.27.0
+    - Added evaluation order option in body agent control (ENT-13295)
+	- Added findlocalgroups policy function (CFE-4550)
+	- Added getgroupinfo policy function (CFE-4512)
+	- Added getgroups policy function (ENT-12722)
+	- Added isnewerthantime policy function (CFE-2815)
+	- Added missing record changes/failures to when flipping the immutable bit
+	  (ENT-13179)
+	- Added policy function classfilterdata() (CFE-3421, ENT-6193)
+	- Added policy function to get ACLs (CFE-4529)
+	- Added sys policy version variable (ENT-4043)
```

Comment:
```suggestion
	- Added sys.policy_version variable (ENT-4043)
```

Reply:
hm, this was already adjusted by @larsewi. I see rebasing went wrong, gotta fix it.